src/cli/commands/distributions/add.ts

```diff
@@ -18,12 +24,14 @@ export const addCommand = new Command("add")
   .description("Add a new distribution")
   .option("-r, --rpc <url>", "RPC endpoint URL. If not provided, RPC_URL environment variable will be used")
   .option("-d, --distributor <address>", "Address of the distributor")
+  .option("-k, --key <privateKey>", "Private key with admin permissions or index to derive from mnemonic. If not provided, PRIVATE_KEY environment variable will be used")
```

Comment:
> Private key with admin permissions 

This is confusing. Private keys have no permissions on their own and it's already shall be clear from context of naming that you need a private key to send tx. 

> or index to derive from mnemonic 

If you want to use mnemonic then use separate -m option and parse it, it's not a good idea to mix stuff, it's confusing.